.storybook/main.js

```diff
@@ -40,6 +42,15 @@ module.exports = {
 
     config.resolve.extensions.push('.ts', '.tsx');
 
+    config.resolve.alias = {
+      ...(config.resolve.alias || {}),
```

Comment:
To make it a convention for the project, also `images` could be added to aliases as it is required like so in stories:
`../../images`